WeatherForecast/WeatherForecastTests/WeatherForecastTests.swift

```diff
@@ -8,25 +8,91 @@ import XCTest
 @testable import WeatherForecast
 
 class WeatherForecastTests: XCTestCase {
-
-    override func setUpWithError() throws {
-        // Put setup code here. This method is called before the invocation of each test method in the class.
+    
+    let parsingManager = ParsingManager()
+    
+    func test_JSON파일인CurrentWeather를_디코딩했을때_id는420006353이다() {
+        //given
+        var outputValue: Int?
+        let expectedValue = 420006353
+        let path = Bundle(for: type(of: self)).path(forResource: "CurrentWeather", ofType: "json")
+        let jsonFile = try! String(contentsOfFile: path!).data(using: .utf8)
+        //when
+        let data = parsingManager.parse(jsonFile!, to: CurrentWeather.self)
+        switch data {
+        case .success(let currentWeather):
+            outputValue = currentWeather.id
+        case .failure(_):
+            outputValue = nil
+        }
+        //then
+        XCTAssertEqual(expectedValue, outputValue)
     }
-
-    override func tearDownWithError() throws {
-        // Put teardown code here. This method is called after the invocation of each test method in the class.
+    
+    func test_JSON파일인FiveDayWeather를_디코딩했을때_id는2643743이다() {
```

Comment:
테스트코드 메서드명이 명세서처럼되려면 이렇게 상세값을 넣어주는 것 보다는 검증하려는 것에 집중해서 네이밍해주면 더 좋을 것 같습니다 ㅎㅎ 

```suggestion
    func testDecode_fiveDayWeather() {
```